aics_pipeline_uploaders/util/celigo.py

```diff
@@ -0,0 +1,52 @@
+from datetime import datetime
+import requests
+
+SERVICE_MMS = "mms"
+MMS_RESOURCE_PLATE_INFO = "plate_info"
+ENV_SERVICE_MAP = {
+    "stg": {
+        SERVICE_MMS: {
+            MMS_RESOURCE_PLATE_INFO: 'http://stg-aics-api.corp.alleninstitute.org/metadata-management-service/1.0/plate/query?barcode='
+        }
+    },
+    "prod": {
+        SERVICE_MMS: {
+            MMS_RESOURCE_PLATE_INFO: 'http://aics-api.corp.alleninstitute.org/metadata-management-service/1.0/plate/query?barcode='
+        }
+    }
+}
+
+class CeligoUtil():
+
+    def __init__(self, env: str = "stg"):
+        self.env = env
+
+    def lookup_well_id(self, plate_barcode, well_name):
+        well_info_response = requests.get(ENV_SERVICE_MAP[self.env][SERVICE_MMS][MMS_RESOURCE_PLATE_INFO] + str(plate_barcode))
+        well_info = well_info_response.json()
+        well_name_lookup = well_info['data'][0]['wellNameLookup']
```

Comment:
We had talked about adding some handling here where multiple imaging sessions had occured. Currently this method just takes the first element of that returned well_info list.